### PR TITLE
Ensure that we show default target group with no args

### PR DESCRIPTION
**Pull request by sidpremkumar**
## Description of the change

Someone pointed out that by default we should show the target groups if no args are passed. This PR implements that logic. 

## Relevant release note information

Release Notes: Ensure that default target group shows by default

## Related JIRA tickets

Relates to JIRA: CWC-1568

## Have you considered the security impacts?

Does this PR have any security impact?

- [ ] Yes
- [X] No

If yes, please explain:

### CODE REVIEW

File: src/handlers/default-target-group/default-target-group.command-builder.ts [hunk 1]
```diff
@@ -1,27 +1,19 @@
...
-                type: 'boolean',
-                alias: 'v',
-                default: false,
-                demandOption: false,
-            }
-        )
         .example('$0 default-targetGroup -g system:masters', 'Set default target group to system:masters')
-        .example('$0 default-targetGroup', 'Set default target group back to none');
+        .example('$0 default-targetGroup -g', 'Reset default target group')
+        .example('$0 default-targetGroup', 'View default target group');
```
Comment (tflpd) [inline src/handlers/default-target-group/default-target-group.command-builder.ts:18]: Adjust examples to new flag

File: src/handlers/default-target-group/default-target-group.command-builder.ts [hunk 2]
```diff
@@ -1,27 +1,19 @@
...
 
-export type defaultTargetGroupArgs = { groups: string[] } &
-    { view: boolean}
+export type defaultTargetGroupArgs = { set: string[] }
 
 export function defaultTargetGroupCmdBuilder(yargs: yargs.Argv<{}>) :
 yargs.Argv<defaultTargetGroupArgs> {
     return yargs
-        .option('groups',
+        .option('set',
```
Comment (tflpd) [inline src/handlers/default-target-group/default-target-group.command-builder.ts:8]: Check with product about this - I agree that this is def better

**Review by tflpd — APPROVED** [review]

File: src/handlers/default-target-group/default-target-group.command-builder.ts (hunk 1 above)
Comment (sidpremkumar) [reply src/handlers/default-target-group/default-target-group.command-builder.ts]: https://github.com/bastionzero/zli/pull/324/commits/58a226fe796d6d509a3c80d84b3ba1813ea57039

**Review by tflpd — APPROVED** [review]

File: src/handlers/default-target-group/default-target-group.command-builder.ts (hunk 2 above)
Comment (sidpremkumar) [reply src/handlers/default-target-group/default-target-group.command-builder.ts:8]: Confirmed w product that this is g2g

**Pull request merged**